terraform/stacks/agha_infra/main.tf

```diff
@@ -83,11 +85,16 @@ resource "aws_s3_bucket_public_access_block" "agha_gdr_staging_2" {
   ignore_public_acls      = true
   restrict_public_buckets = true
 }
+resource "aws_s3_bucket_ownership_controls" "agha_gdr_staging_2" {
```

Comment:
Why do we need this? 
(I think it's against the best practises)

Reply:
With the current deployment, staging and results bucket have the `Bucket owner enforced` but the store bucket still have the `Object writer`. So to enable this at the store bucket, I enabled this here, but I added for all 3 buckets for consistency.

I think the default/recommendation is to make `bucket-owner-enforced` and it seems that not specifying this will just not touch what is currently deployed.

Reply:
OK, thanks!
I'm happy to have it explicit if it enforces the bucket owner. 
I am just trying to remember why the store is set to object writer... especially on the store bucket I think we want to be the ones having the full ownership, no?

Reply:
Yeah I think we should have the ownership. Andrew made a [comment](https://umccr.slack.com/archives/CCBADM4SW/p1698380172962759) on this last month.

Reply:
Hm, there must have been a reason why we switched, but I can't remember now...
OK, then let's switch it back to owner enforced. We'll notice if something breaks... sooner or later... ;-)